frontend/src/components/features/home/home-header.tsx

```diff
@@ -18,22 +19,37 @@ export function HomeHeader() {
   const isCreatingConversation =
     isPending || isSuccess || isCreatingConversationElsewhere;
 
+  // Note: handleCreateConversation is now implemented inline in the NavLink onClick
+
   return (
     <header className="flex flex-col gap-5">
       <AllHandsLogo />
 
       <div className="flex items-center justify-between">
         <h1 className="heading">{t("HOME$LETS_START_BUILDING")}</h1>
-        <BrandButton
-          testId="header-launch-button"
-          variant="primary"
-          type="button"
-          onClick={() => createConversation({ conversation_trigger: "gui" })}
-          isDisabled={isCreatingConversation}
+        <NavLink
+          to="/"
+          onClick={(e) => {
+            // If cmd/ctrl key is pressed, let the default behavior happen (open in new tab)
+            if (e.metaKey || e.ctrlKey) {
```

Comment:
This feels like relying on keymap convention rather than the semantic meaning of the action, but I don't have an alternative right now so cool beans